include/deal.II/lac/cuda_sparse_matrix.h

```diff
@@ -340,6 +399,131 @@ namespace CUDAWrappers
   {
     return nnz;
   }
+
+
+
+  template <typename Number>
+  template <class StreamType>
+  inline void
+  SparseMatrix<Number>::print(StreamType &out,
```

Comment:
Have you checked that the function prints the matrix correctly?

Reply:
Yes, but let me write a test.